src/app/components/Error404Page/Error404Page.jsx

```diff
@@ -6,9 +6,15 @@ const Error404Page = () => (
     <p>
       We’re sorry. The page you were looking for doesn’t exist.
     </p>
+    <p>
+      Ready to discover your next great read? Browse and filter our&nbsp;
+      <a href="/books-music-movies/recommendations/best-books/staff-picks">
+        Staff Picks.
+      </a>
+    </p>
     <p>
       Need help or have a question?&nbsp;
-      <a href="https://www.nypl.org/get-help/contact-us">
+      <a href="/get-help/contact-us">
```

Comment:
Just fyi, this won't work in the `staff-picks-development.....com` domains but will work fine in the reverse proxy app.

Reply:
My attempt to avoid adding absolute links to apps. If this is not wise, let me know if I should revert.

Reply:
This is fine but we'll probably get some "it's not working"  feedback if anyone tests the non reverse proxy urls.